File: javascript/9_pythagorean.js


for(var a= 2; a <= 333; a++){
  for(var b = a+1; b < (1000-a)/2; b++){
    var c = 1000 -a -b;

    if(Math.pow(a,2) + Math.pow(b,2) == Math.pow(c,2)){
      console.log(a*b*c);
      break;
    }

  }
}

File: C++/9_pythagorean.cpp
#include <iostream>
#include <cmath>

using namespace std;
int main(){

  for(int a=0; a <= 333; a++){
    for(int b=a+1; b < (1000-a)/2; b++){
      int c = 1000 -a-b;

      if(pow(a,2) + pow(b,2) == pow(c,2)){
        cout << a*b*c << endl;
      }
    }
  }


  return 0;
}